lib/html_renderer.rb

```diff
@@ -0,0 +1,16 @@
+class HtmlRenderer
+  def render(json_result)
```

Comment:
Please notice that `HtmlRenderer` is in the default namespece. 

Since runners may also be deployed using corso, that may be a problem, since it can collide with potential `HtmlRenderer`s in other runners. 

I would name it `QsimHtmlRenderer` or use a namespace. In this case, I prefer the latter. 

That may collide